#Message type to allow the robots to constantly share the edges they have already visited
int16 sender
int16[] destinations
int16[] forbiden_Edges

